src/components/NavBar/NavBar.css

```diff
@@ -1,48 +1,48 @@
 .ant-menu-submenu-open.ant-menu-submenu- {
-	color: #657a3a;
+	color: #839461;
 }
 
 .ant-menu.ant-menu-light.ant-menu-root.ant-menu-horizontal {
 	width: 100%;
 }
 
 .ant-menu-submenu.ant-menu-submenu-horizontal {
-	left: 30%;
+	left: 31%;
 }
 .ant-menu-item.ant-menu-item-active {
-	color: #657a3a;
+	color: #839461;
```

Comment:
I'm using `#657a3a` as the font-color variable in the new theming. Might need to change?